src/main/setupInvestHandlers.js

```diff
@@ -156,14 +167,20 @@ export function setupInvestRunHandlers(investExe) {
 
     investRun.on('exit', (code) => {
       delete runningJobs[jobID];
-      event.reply(`invest-exit-${jobID}`, code);
+      event.reply(`invest-exit-${jobID}`, {
+        code: code,
+        stdErr: investStdErr,
```

Comment:
Okay, so we can just bundle that stderr reply in here instead of having it be a separate call in the Callback itself.

Reply:
Yes, I noticed there was no need to send each stderr message individually to the renderer as they stream in. So we cut down on some ipc traffic by just sending the complete stderr at the end.